app/views/historic_appointments/show.html.erb

```diff
@@ -0,0 +1,67 @@
+<% content_for :title, "History of #{@historic_appointment.title}" %>
+<% page_class "govuk-width-container" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/title", {
+      context: I18n.t("historic_appointment.context"),
+      title: @historic_appointment.title,
+    } %>
+
+    <%= render "govuk_publishing_components/components/lead_paragraph", {
+      text: "#{@historic_appointment.political_party} #{@historic_appointment.dates_in_office}",
+    } %>
+    <%= render "govuk_publishing_components/components/lead_paragraph", {
+      text: @historic_appointment.description,
+    } %>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <hr class="govuk-section-break govuk-section-break--s govuk-section-break--visible govuk-!-margin-bottom-4">
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-third">
+        <%= image_tag(
+          @historic_appointment.image_data["url"], 
+          alt: @historic_appointment.image_data["alt_text"], 
+          class: "govuk-!-margin-bottom-4 govuk-!-width-full"
+        ) %>
+      </div>
+    </div>
+
+    <% @historic_appointment.appointment_info_array.each do |info| %>
+      <% if info[:text].present? %>
+        <%= render "govuk_publishing_components/components/heading", {
+          text: info[:title],
+          font_size: "m",
+          margin_bottom: 1,
+        } %>
+        <p class="govuk-body"><%= info[:text] %></p>
+      <% end %>
+    <% end %>
+
+    <%=  render "govuk_publishing_components/components/heading", {
+      text: I18n.t("historic_appointment.headings.biography"),
+      font_size: "m",
+      margin_bottom: 1,
+    } %>
+
+    <%=  render "govuk_publishing_components/components/govspeak" do %>
+      <%=  @historic_appointment.biography %>
+    <%  end %>
+
+  </div>
+
+  <div class="govuk-grid-column-one-third">
+    <p class="govuk-body">
+      <%=  render "govuk_publishing_components/components/related_navigation", {
+      content_item: @historic_appointment.related_historic_appointments_nav,
+      ga4_tracking: true
+    } %>
+    <p class="govuk-body">
+      <a class="govuk-link" href="/government/history/past-prime-ministers">View all past prime ministers</a>
```

Comment:
Looks like there's another bit of hardcoded text in this link and also in the title on line 1.

Reply:
Should all be gone now, did this and also moved the page title into a method on the model.